frontend/common/types/responses.ts

```diff
@@ -295,6 +295,18 @@ export type Account = {
   is_superuser: boolean
 }
 
+export type Role = {
+  id?: number
```

Comment:
Are all of these optional ?

Reply:
According to the API yes
<img width="1268" alt="Screen Shot 2023-11-01 at 10 34 10" src="https://github.com/Flagsmith/flagsmith/assets/41410593/f03bc0e9-9936-4098-8f8f-f3c34fae1de8">
but I see that they should not be optional here.
Corrected.